cupy/lib/_shape_base.py

```diff
@@ -76,7 +76,7 @@ def _make_along_axis_idx(arr_shape, indices, axis):
     dest_dims = list(range(axis)) + [None] + \
         list(range(axis + 1, indices.ndim))
 
-    # build a fancy index, consisting of orthogonal aranges, with the
+    # build a fancy index, consisting of orthogonal arranges, with the
```

Comment:
This change should not be made. "aranges" is refering to multiple calls to `cupy.arange` in the loop below.

Reply:
One option could be to refer to them as just "orthogonal ranges".  One might associate "range" with Python's built-in `range` rather than `cupy.arange`, but I think the meaning would still be clear enough given the actual code within the loop.

Reply:
`orthogonal cupy.arange calls`